index.js

```diff
@@ -109,7 +119,8 @@ require('util').inherits(RingPop, EventEmitter);
 
 RingPop.prototype.destroy = function destroy() {
     this.destroyed = true;
-    clearTimeout(this.gossipTimer);
+    this.gossip.stop();
```

Comment:
i don't like `destroy()` because it has a sense of finality to it.in this case, with the gossip object, a `stop()` can be `start()`'ed again. calling destroy() and then start() again would be weird.